filebeat/prospector/prospector.go

```diff
@@ -77,9 +74,9 @@ func (p *Prospector) initProspectorer(outlet channel.Outleter, states []file.Sta
 	case cfg.StdinInputType:
 		prospectorer, err = stdin.NewProspector(config, outlet)
 	case cfg.LogInputType:
-		prospectorer, err = NewLog(config, states, p.registry, outlet, p.done)
+		prospectorer, err = log.NewProspector(config, states, outlet, p.done)
 	default:
-		return fmt.Errorf("Invalid input type: %v", p.config.InputType)
+		return fmt.Errorf("invalid prospector type: %v. Change input_type.", p.config.InputType)
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
error strings should not be capitalized or end with punctuation or a newline